resources/assets/js/components/pages/home/NewSpot.vue

```diff
@@ -336,8 +340,12 @@
             },
             parseDescriptors() {
                 let descriptorIDtoValue = {};
-                this.requiredDescriptors.forEach((descriptor)=>{
-                    descriptorIDtoValue[descriptor.id] = this.spotDescriptors[descriptor.name];
+                this.requiredDescriptors.forEach((descriptor) => {
+                    let descriptorValue = this.spotDescriptors[descriptor.name];
+                    descriptorIDtoValue[descriptor.id] =
+                        typeof descriptorValue === 'object' ?
```

Comment:
I think you should use `Array.isArray()` here as it also protects against the possibility of trying to call `join()` on an object.

Reply:
Good catch! Thank you